docs/technical-documentation.md

```diff
@@ -1,54 +1,66 @@
 # Technical documentation
+
 ## Content
+
 * [Glossary](16-glossary.md)
```

Comment:
actually i cannot find this file. Am I missing something?

Reply:
Hmm... probably this was removed when migrating the glossary to the wiki? @maxammann ?
Maybe we can replace it with a link to https://wiki.integreat-app.de/glossary